actiontext/test/unit/content_test.rb

```diff
@@ -164,6 +164,12 @@ class ActionText::ContentTest < ActiveSupport::TestCase
     assert_equal trix_html, content_from_html(html).to_trix_html.strip
   end
 
+  test "sanitizes attachment markup for Trix if content exists but is empty" do
+    html = '<action-text-attachment sgid="123" content=""></action-text-attachment>'
+    trix_html = '<figure data-trix-attachment="{&quot;sgid&quot;:&quot;123&quot;}"></figure>'
+    assert_equal trix_html, content_from_html(html).to_trix_html.strip
+  end
+
   test "does not add missing content attribute" do
```

Comment:
Maybe this test is redundant now?

Reply:
@p8 I'm not entirely sure. On the one hand it does seem to test for the same thing as the test where we have `content=""`, but on the other hand maybe it's good to test "missing" and "empty" as separate cases?

Anyway, sorry for the delay, but I just merged your other suggestions and did a little clean up as a result of the merge. Please let me know if you want me to remove the possibly-redundant test.